### PR TITLE
Refactor/blockbase color admin

### PR DESCRIPTION
**Pull request by pbking**
#### Changes proposed in this Pull Request:

* Removes the Color Customization controls in the Customizer.
* Adds the "Palette Options" previously available in the Customizer Color Customization as "Theme Variations"

image

image

Note: Variations do NOT inherit from the parent.  This means that any child theme that does NOT provide theme variations will not have Blockbase's.  Child themes that did NOT provide palette collections would have been provided Blockbase's in the Customizer.

This change should not BREAK any themes, however any child themes should migrate their palettes to variations to continue to provide that functionality.

All of the public (and Videomaker) blockbase themes have been refactored (Geologist, Quadrat, Zoologist and Videomaker)

#### Related issue(s):

NOTE: This should be merged and released at the same time as the Font Customization refactor and released as a Major Point Release for the Blockbase theme.

### CODE REVIEW

File: blockbase/styles/featured-one.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+	"version": 2,
+	"title": "Featured One",
```
Comment (MaggieCabrera) [inline blockbase/styles/featured-one.json]: @beafialho should we rename these color palettes/variations?

Comment (beafialho) [reply blockbase/styles/featured-one.json]: Are these the Videomaker ones?

Comment (pbking) [reply blockbase/styles/featured-one.json]: These are the Blockbase color "schemes" from the customizer converted to theme variations.  They didn't have names in Blockbase, just pictures and the "featured" label and I used the names that we had given them in configuration.  As user-surfaced names they are certainly lack-luster.

image

Comment (beafialho) [reply blockbase/styles/featured-one.json]: Oh, I see! Here's what I suggest then:

Default > **Ocean Breeze**
Featured (Red) > **Ruby Wine**
Featured (Green) > **Rainforest**
Featured (Pastel Colors) > **Charcoal**

Comment (pbking) [reply blockbase/styles/featured-one.json]: Thanks!  Nice names.  :)

Comment (pbking) [reply blockbase/styles/featured-one.json]: image
image

File: blockbase/inc/customizer/wp-customize-colors.php [hunk 2]
```diff
@@ -98,87 +19,19 @@ function register_color_controls( $wp_customize, $palette ) {
...
-				$setting_key,
-				array(
-					'section' => $this->section_key,
-					'label'   => $palette_item['name'],
-				)
+			$this->section_key . '-migration-notice',
+			array(
+				'type'        => 'hidden',
+				'description' => '<div class="notice notice-warning">
+				<p>' . __( 'Color customization is now available exclusively in the Full Site Editor Global Styles panel.  (Colors>Palette>⋮>Edit Colors) <br><br>Featured Palettes can now be found in the Full Site Editor as Theme Variations. (Browse Styles)', 'blockbase' ) . '</p>
```
Comment (mikachan) [inline blockbase/inc/customizer/wp-customize-colors.php]: This is a tiny nitpick, but should there be a colon (:>) here? `(Colors>Palette>⋮>Edit Colors)`

**Review by mikachan — APPROVED** [review]
This is working well for me. I can't see any errors on Blockbase or any child themes. I can see all the color palettes have been turned into variations. Switching between themes and variations is working really well on my side, looks good to go! ✨

File: blockbase/styles/featured-one.json (hunk 1 above)
Comment (beafialho) [reply blockbase/styles/featured-one.json]: Thanks! Perhaps that's already showing but "Default" show "Ocean Breeze" instead.

File: blockbase/inc/customizer/wp-customize-colors.php (hunk 2 above)
Comment (pbking) [reply blockbase/inc/customizer/wp-customize-colors.php]: No, it's supposed to be the three dots menu, not a colon.  I wasn't sure how else to describe that. :P

image

Comment (mikachan) [reply blockbase/inc/customizer/wp-customize-colors.php]: Ooh yeah! Hmm, I'm not sure of a better way to describe it either.

Comment (madhusudhand) [reply blockbase/inc/customizer/wp-customize-colors.php]: there is a unicode character ⋮

**Comment by pbking** [comment]
Closing: Work merged in #6167

**Pull request closed without merging**